nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -39,6 +39,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- The `pam_ssh_agent_auth(8)` module now trusts files listed in `security.pam.sshAgentAuth.authorizedKeysFiles`,
+  defaulting to `/etc/ssh/authorized_keys.d/%u` only; previously, the set of trusted `authorized_keys` files wasn't
+  configurable and included `~/.ssh/authorized_keys` **which is insecure**:
+  see [#31611](https://github.com/NixOS/nixpkgs/issues/31611).
+
```

Comment:
On one hand, this note could make it more obvious that upgrading could cause a user to lose access to anything protected by `pam_ssh_agent_auth`. On the other hand, they could always boot up an earlier configuration.